distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -279,6 +183,9 @@ type Array
     insert : Integer -> Any -> Vector ! Index_Out_Of_Bounds
     insert self at=self.length item=Nothing = Vector.insert self at item
 
+    ## PRIVATE
+    insert_builtin self at values = @Builtin_Method "Vector.insert_builtin"
```

Comment:
What is the reason for `insert_builtin` to be defined on `Array` when nobody call it?